src/modules/launcher/Plugins/Microsoft.Plugin.Calculator.UnitTest/ExtendedCalculatorParserTests.cs

```diff
@@ -110,7 +112,15 @@ public void Interpret_DifferentCulture_WhenCalled(string input, decimal expected
         [TestCase("((1 * 2)", false)]
         [TestCase("(1 * 2)))", false)]
         [TestCase("abcde", false)]
-        [TestCase("plot( 2 * 3)", true)]
```

Comment:
Why is this test case deleted?

Reply:
`plot` is not a valid input to the mages library and would throw an exception. Hence, it has been removed from the Regex which checks if the input is valid for the calculator plugin. Since any functionality can be added in the future I did not think it was right to change the output of the unit test from true to false simply because it is not supported right now. Hence, I removed the test.